cpp/cmake/bbp-setup-pre-commit-config.py

```diff
@@ -36,6 +36,12 @@ def _parse_cli(args=None):
         default=False,
         nargs="?",
     )
+    parser.add_argument(
+        "--regenerate-hooks",
```

Comment:
What about `-f/--force` instead of `--regenerate-hooks` ?

Reply:
ah yes, that was obvious :) fixed.